apps/dashboard/src/contract-ui/tabs/listings/components/listing-stats.tsx

```diff
@@ -1,87 +1,90 @@
 import { Skeleton, Stack, Stat, StatLabel, StatNumber } from "@chakra-ui/react";
-import {
-  useDirectListingsCount,
-  useEnglishAuctionsCount,
-} from "@thirdweb-dev/react";
-import type { MarketplaceV3 } from "@thirdweb-dev/sdk";
-import { BigNumber } from "ethers";
 import { useMemo } from "react";
+import type { ThirdwebContract } from "thirdweb";
+import { totalAuctions, totalListings } from "thirdweb/extensions/marketplace";
+import { useReadContract } from "thirdweb/react";
 import { Card } from "tw-components";
 
-interface ListingStatsV3Props {
-  contract?: MarketplaceV3;
-  features: string[];
-}
-
-const TotalListingsStat: React.FC<{ contract?: MarketplaceV3 }> = ({
+const TotalListingsStat: React.FC<{ contract: ThirdwebContract }> = ({
   contract,
 }) => {
-  const directListingsQuery = useDirectListingsCount(contract);
-  const englishAuctionsQuery = useEnglishAuctionsCount(contract);
-
-  const totalListings = useMemo(
-    () =>
-      BigNumber.from(directListingsQuery?.data || 0).add(
-        BigNumber.from(englishAuctionsQuery?.data || 0),
-      ),
-    [directListingsQuery?.data, englishAuctionsQuery?.data],
+  const { data: directListingCount, isSuccess: listingLoaded } =
+    useReadContract(totalListings, {
+      contract,
+    });
+  const { data: englishAuctionCount, isSuccess: auctionLoaded } =
+    useReadContract(totalAuctions, {
+      contract,
+    });
```

Comment:
prefer not destructuring these here, then you don't have to rename every key and can instead just access them by `xQuery.foo` wherever u need